Assets/HoloToolkit-Examples/UX/Scripts/ExampleData/NoteDataProvider.cs

```diff
@@ -46,9 +46,12 @@ public void NoteTypeButtonSelected()
                 return;
             }
             int interactivePos = SourceSet.SelectedIndices[0];
-            string title = SourceSet.Interactives[interactivePos].gameObject.GetComponent<LabelTheme>().Default;
-            TargetGroup.Titles = Data[title];
-            TargetGroup.UpdateData();
+            LabelTheme label = SourceSet.Interactives[interactivePos].gameObject.GetComponent<LabelTheme>();
+            if (label != null)
+            {
+                TargetGroup.Titles = Data[label.Default];
+                TargetGroup.UpdateData();
+            }
```

Comment:
Should we add a warning if our object doesn't have a `LabelTheme` telling our dev to add one?

Reply:
Yes. The Dev would most likely implement his own "DataProvider", where he gets some data from an external source and than acts on it accordingly, putting that data into InteractiveSets/InteractiveGroups as he likes. So I also added some documentation that descibes it a bit more.